packages/util-retry/src/StandardRetryStrategy.ts

```diff
@@ -1,47 +1,88 @@
 import { Provider, RetryErrorInfo, RetryErrorType, RetryStrategyV2, StandardRetryToken } from "@aws-sdk/types";
 
 import { DEFAULT_MAX_ATTEMPTS, RETRY_MODES } from "./config";
-import { DEFAULT_RETRY_DELAY_BASE, INITIAL_RETRY_TOKENS } from "./constants";
-import { getDefaultRetryToken } from "./defaultRetryToken";
+import {
+  DEFAULT_RETRY_DELAY_BASE,
+  INITIAL_RETRY_TOKENS,
+  NO_RETRY_INCREMENT,
+  RETRY_COST,
+  THROTTLING_RETRY_DELAY_BASE,
+  TIMEOUT_RETRY_COST,
+} from "./constants";
+import { getDefaultRetryBackoffStrategy } from "./defaultRetryBackoffStrategy";
+import { createDefaultRetryToken } from "./defaultRetryToken";
 
 /**
  * @public
  */
 export class StandardRetryStrategy implements RetryStrategyV2 {
   public readonly mode: string = RETRY_MODES.STANDARD;
-  private retryToken: StandardRetryToken;
+  private capacity: number = INITIAL_RETRY_TOKENS;
+  private readonly retryBackoffStrategy = getDefaultRetryBackoffStrategy();
```

Comment:
nit: The `retryBackoffStrategy` is not required to be a class member, as it's only used in refreshRetryTokenForRetry
```suggestion
  private readonly retryBackoffStrategy = getDefaultRetryBackoffStrategy();
```

Reply:
we don't need to regenerate the object every method call though